frontend/src/components/footer/footer.jsx

```diff
@@ -5,46 +5,47 @@ import Link from "next/link";
-						<a href="#" target="_blank">
-							<Image src="/images/linkedinIconWhite.svg" width="26" height="26" alt="link to our linkedin"/>
-						</a>
-					</div>
-				</div>
-				<div className={styles.rightSide}>
-					{navigationItems.map(item => (
-						<Link key={item.pathname} href={`/${item.pathname}`}>
-							<p className={styles.linkItem}>{item.name}</p>
-						</Link>
-					))}
-				</div>
-			</section>
-			<section className={styles.copyrightsSection}>
-				<p className={styles.leftSide}>
-					Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa quia nisi ducimus accusamus vitae qui a debitis
-					inventore?
-				</p>
-				<p className={styles.rightSide}>
-					LOREM IPSUM
-				</p>
-			</section>
-		</footer>
-	);
+  return (
+    <footer className={[styles.footer, isHidden ? styles.hidden : ""].join(" ")}>
+      <section className={styles.logoSection}>
+        <img src="/images/logo.white.svg" alt="Negovatelji logo" className={styles.logo} />
+      </section>
+      <section className={styles.mainSection}>
+        <div className={styles.leftSide}>
+          <p>
+            Društvo koje ceni, podržava i osnažuje porodične negovatelje željena je budućnost Centra HORIZONT 21 i
+            platforme NEGOVATELJI.RS kreirane uz podršku podršku VEGAT IT kompanije i projekta Code For Couse.
+          </p>
+          <div className={styles.socialMediaButtons}>
+            <a href="https://www.facebook.com/groups/270285275225942/" target="_blank">
+              <Image src="/images/facebookIconWhite.svg" width="16.11" height="30" alt="link to our facebook" />
+            </a>
+            <a href="https://www.youtube.com/channel/UCS5gh6FEudKxWeUPIzL_Kog" target="_blank">
+              <Image src="/images/youtubeWhite.svg" width="30" height="30" alt="link to our youtube" />
+            </a>
+            <a href="https://instagram.com/negovatelji?igshid=MWI4MTIyMDE=" target="_blank">
+              <Image src="/images/instagramIconWhite.svg" width="25" height="30" alt="link to our instagram" />
+            </a>
+          </div>
+        </div>
+        <div className={styles.rightSide}>
+          {navigationItems.map((item) => (
+            <Link key={item.pathname} href={`/${item.pathname}`}>
+              <p className={styles.linkItem}>{item.name}</p>
+            </Link>
+          ))}
+          <Link key={"PrivacyPolicy"} href={"/documents/privacyPolicy.pdf"}>
+            <p className={styles.linkItem}>Politika privatnosti</p>
+          </Link>
+          <Link key={"TermsAndConditions"} href={"/documents/termsAndConditions.pdf"}>
+            <p className={styles.linkItem}>Uslovi korišćena sajta</p>
+          </Link>
+        </div>
+      </section>
+      <section className={styles.copyrightsSection}>
+        <p className={styles.leftSide}>© Copyright 2022 Negovatelji.rs - Sva prava zadržana</p>
+        <p className={styles.rightSide}>Designed by VEGA IT</p>
+      </section>
+    </footer>
+  );

```

Comment:
Vidim da nema nikakvih izmena u `.scss` fajlovima... 
U dokumentu u kojem se nalaze novi zahtevi, pise da linkove treba raspodeliti u 2 kolone, a nas trenutni kod dozvoljava da se ti linkovi raspodele i u 4 kolone (u zavisnosti od sirine ekrana). 
Najmanji broj kolona koje nas trenutni kod dozvoljava je 2 i treba samo taj broj iz dinamickog pretvoriti u fiksni, tj. da uvek bude 2 kolone bez obzira kolika je sirina ekrana. 
Mozes i u potpunosti izmeniti SCSS koji se odnosi na taj deo footer-a, sve dok odgovara tim novim zahtevima.
Ti novi zahtevi su: 2 kolone, gde svaka kolona ima navedene item-e koje u sebi treba da ima i redosled tih item-a. 
To znaci da neces moci koristiti `navigationItems.map`. 
Jedno od resenja bi bilo da svaki item u `navigationItems` izdvojis u zaseban objekat i reference thih objekata ubacis u `navigationItems` listu. 
Primer item objekta: 
```
export const HOME_NAV_ITEM = {
		name: "Zajednica",
		pathname: HOME_PATHNAME,
		active: true,
	}
```
Refaktorisana `navigationItems` lista:
``` 
export const navigationItems = [
    HOME_NAV_ITEM,
    CAREGIVERS_NAV_ITEM,
    ...
]
```
Ovo ti omogucava da u `footer.jsx` (ili bilo kojem drugom fajlu) kreiras specificnu listu nav item-a, sa drugacijim brojem i redosledom. U tvom slucaju, trebaju ti dve liste (leva i destna kolona).